kolibri/plugins/coach/assets/src/state/getters.js

```diff
@@ -1,4 +1,4 @@
-const Constants = require('../constants');
+const ReportConstants = require('../reportConstants');
```

Comment:
Was thinking the stuff in this file might be more appropriate for the view?

Also started thinking about how `learner` and `class` are just 2 different representations of the same view, so you might not need a component for both. Thoughts?